File: android/sslcommerzlibrary/src/main/java/com/sslwireless/sslcommerzlibrary/model/initializer/SSLCCustomerInfoInitializer.java
package com.sslwireless.sslcommerzlibrary.model.initializer;

public class SSLCCustomerInfoInitializer {
    private String customerName;
    private String customerEmail;
    private String customerAddress1;
    private String customerAddress2;
    private String customerCity;
    private String customerState;
    private String customerPostCode;
    private String customerCountry;
    private String customerPhone;
    private String customerFax;

    public SSLCCustomerInfoInitializer(String customerName, String customerEmail,
                                       String customerAddress1,
                                       String customerCity,
                                       String customerPostCode, String customerCountry,
                                       String customerPhone) {
        this.customerName = customerName;
        this.customerEmail = customerEmail;
        this.customerAddress1 = customerAddress1;
        this.customerCity = customerCity;
        this.customerPostCode = customerPostCode;
        this.customerCountry = customerCountry;
        this.customerPhone = customerPhone;
    }

    public SSLCCustomerInfoInitializer addAddress2(String customerAddress2) {
        this.customerAddress2 = customerAddress2;
        return this;
    }

    public SSLCCustomerInfoInitializer addState(String customerState) {
        this.customerState = customerState;
        return this;
    }

    public SSLCCustomerInfoInitializer addFax(String customerFax) {
        this.customerFax = customerFax;
        return this;
    }

    public String getCustomerName() {
        return customerName;
    }

    public void setCustomerName(String customerName) {
        this.customerName = customerName;
    }

    public String getCustomerEmail() {
        return customerEmail;
    }

    public void setCustomerEmail(String customerEmail) {
        this.customerEmail = customerEmail;
    }

    public String getCustomerAddress1() {
        return customerAddress1;
    }

    public void setCustomerAddress1(String customerAddress1) {
        this.customerAddress1 = customerAddress1;
    }

    public String getCustomerAddress2() {
        return customerAddress2;
    }

    public void setCustomerAddress2(String customerAddress2) {
        this.customerAddress2 = customerAddress2;
    }

    public String getCustomerCity() {
        return customerCity;
    }

    public void setCustomerCity(String customerCity) {
        this.customerCity = customerCity;
    }

    public String getCustomerState() {
        return customerState;
    }

    public void setCustomerState(String customerState) {
        this.customerState = customerState;
    }

    public String getCustomerPostCode() {
        return customerPostCode;
    }

    public void setCustomerPostCode(String customerPostCode) {
        this.customerPostCode = customerPostCode;
    }

    public String getCustomerCountry() {
        return customerCountry;
    }

    public void setCustomerCountry(String customerCountry) {
        this.customerCountry = customerCountry;
    }

    public String getCustomerPhone() {
        return customerPhone;
    }

    public void setCustomerPhone(String customerPhone) {
        this.customerPhone = customerPhone;
    }

    public String getCustomerFax() {
        return customerFax;
    }

    public void setCustomerFax(String customerFax) {
        this.customerFax = customerFax;
    }

}

File: lib/model/SSLCCustomerInfoInitializer.dart
import 'package:flutter/material.dart';
import 'package:json_annotation/json_annotation.dart';

part 'SSLCCustomerInfoInitializer.g.dart';

@JsonSerializable()
class SSLCCustomerInfoInitializer {
  String customerName;
  String customerEmail;
  String customerAddress1;
  String customerAddress2;
  String customerCity;
  String customerState;
  String customerPostCode;
  String customerCountry;
  String customerPhone;
  String customerFax;

  SSLCCustomerInfoInitializer(
      {@required this.customerName,
      @required this.customerEmail,
      @required this.customerAddress1,
      this.customerAddress2,
      @required this.customerCity,
      this.customerState,
      @required this.customerPostCode,
      @required this.customerCountry,
      @required this.customerPhone,
      this.customerFax});

  Map<String, dynamic> toJson() => _$SSLCCustomerInfoInitializerToJson(this);

  factory SSLCCustomerInfoInitializer.fromJson(Map<String, dynamic> json) =>
      _$SSLCCustomerInfoInitializerFromJson(json);

  SSLCCustomerInfoInitializer addAddress2(String customerAddress2) {
    this.customerAddress2 = customerAddress2;
    return this;
  }

  SSLCCustomerInfoInitializer addState(String customerState) {
    this.customerState = customerState;
    return this;
  }

  SSLCCustomerInfoInitializer addFax(String customerFax) {
    this.customerFax = customerFax;
    return this;
  }

  String getCustomerName() {
    return this.customerName;
  }

  void setCustomerName(String customerName) {
    this.customerName = customerName;
  }

  String getCustomerEmail() {
    return this.customerEmail;
  }

  void setCustomerEmail(String customerEmail) {
    this.customerEmail = customerEmail;
  }

  String getCustomerAddress1() {
    return this.customerAddress1;
  }

  void setCustomerAddress1(String customerAddress1) {
    this.customerAddress1 = customerAddress1;
  }

  String getCustomerAddress2() {
    return this.customerAddress2;
  }

  void setCustomerAddress2(String customerAddress2) {
    this.customerAddress2 = customerAddress2;
  }

  String getCustomerCity() {
    return this.customerCity;
  }

  void setCustomerCity(String customerCity) {
    this.customerCity = customerCity;
  }

  String getCustomerState() {
    return this.customerState;
  }

  void setCustomerState(String customerState) {
    this.customerState = customerState;
  }

  String getCustomerPostCode() {
    return this.customerPostCode;
  }

  void setCustomerPostCode(String customerPostCode) {
    this.customerPostCode = customerPostCode;
  }

  String getCustomerCountry() {
    return this.customerCountry;
  }

  void setCustomerCountry(String customerCountry) {
    this.customerCountry = customerCountry;
  }

  String getCustomerPhone() {
    return this.customerPhone;
  }

  void setCustomerPhone(String customerPhone) {
    this.customerPhone = customerPhone;
  }

  String getCustomerFax() {
    return this.customerFax;
  }

  void setCustomerFax(String customerFax) {
    this.customerFax = customerFax;
  }
}
